app/controllers/names/maps_controller.rb

```diff
@@ -12,6 +12,7 @@ def show
       return unless @name
 
       @query = create_query(:Observation, names: @name.id)
+      @query.save
```

Comment:
Check `ApplicationController::Queries#find_or_create_query(model_symbol, args = {})`

This seems to be what other controllers are calling instead of these two lines.

Reply:
Yes, that's what `controllers/locations/maps_controller.rb` is doing. 
I think that's the way to go here for now, until we revisit these methods.